src/Components/maps.js

```diff
@@ -128,32 +178,286 @@ class UgMap extends Component {
         if (sliderK === "ppp_sum") {
           this.PopulationNationalGridValue += val;
         }
+        if (
```

Comment:
This section of code that you have added is not scalable or maintainable. By scalable I mean that if more features are added, this implementation will become large and difficult to monitor. I advise that you employ javascript features to create something simpler. Not going to be able to merge it in otherwise

Reply:
@RonKbS , okay i understand. i will try breaking down the code.